### ISSUE TITLE
Icons are not displaying

### ISSUE THREAD

**Issue by raghavakhil** [komarovalexander/ka-table#193]
![image](https://user-images.githubusercontent.com/40756706/148720277-7b2a69c2-c582-458e-987d-6a6257483582.png)

Sort and Filter Icons are not displaying in ka-table v 7.3.1. Any advise to resolve this issue? Please let me know

**Comment by komarovalexander** [comment]
Hi @raghavakhil is it real project or is this stackblitz?

**Comment by raghavakhil** [comment]
This is a stackblitz example, but even when I tried in real project I am not able to see the icons

And it is better to enhance the filter with "select all" option by adding another checkbox at the start of the filter values

**Comment by komarovalexander** [comment]
@raghavakhil can you share the zip file with the project where it does not work?

**Comment by raghavakhil** [comment]
Sorry I cannot share that one....

**Comment by zatopek-firstis** [comment]
Greetings, I have ran into the same problem with icons. 

By what I have found, the problem has 2 possible causes. 
First cause is by overwriting font used in KaTable, and that particular font does not have support for utf-8 (unicode) icons.
Second cause is when client computer does not have the used font with support for icons installed.

Unfortunately we have to use our font, which does not support such icons.

I was able to bypass this problem by overwriting cell in childComponents, where I copied everything except of said icons, there I used font-awesome ones.

Here is how I did it, feel free to copy and customize to your needs:
![tree-expand-icons-bypass](https://user-images.githubusercontent.com/72503723/198966316-cbdab23d-611a-4755-aacc-4552456213ba.png)

Note that this solution is for icons in Tree Mode, and I am influencing only the first cell, where said icons should appear.
I also had to overwrite css for ka-tree-empty-space from 20px to 9px, because it caused more space than with original icons.

I do not guarantee for any undesirable side-effects caused by my solution. There may be some.

**Comment by normand1** [comment]
Yep, turned out using a font that didn't support the unicode icons was my issue as well. Solved my issue by updating my css like so:
```
span.ka-icon.ka-icon-sort.ka-icon-sort-arrow-up {
  font-family: 'icons' !important;
}

span.ka-icon.ka-icon-sort.ka-icon-sort-arrow-down {
  font-family: 'icons' !important;
}
```

**Comment by komarovalexander** [comment]
Hi If somebody faced with that issue again to workaround you also can try to import styles directly 
```
@import 'ka-table/static/icons.scss';
```
the demo: https://stackblitz.com/edit/table-sorting-ts-qsrrzn?file=style.scss

**Comment by komarovalexander** [comment]
the icons issue has been fixed starting from `v8.0.0`, you fo not need `@import 'ka-table/static/icons.scss';` and this is not available anymore
